rmgpy/cantherm/input.py

```diff
@@ -234,7 +246,7 @@ def transitionState(label, *args, **kwargs):
         job = StatMechJob(species=ts, path=path)
         jobList.append(job)
     
-    elif len(args) == 0 and len(kwargs) > 0:
```

Comment:
If no QM or direct input is given, I think the proper behavior should be to throw an error (since the user is not giving cantherm enough information).

Reply:
Is there a reason the condition here differs from the respective condition is `species`?
In other words, shouldn't this be `if len(kwargs) > 0` instead of `elif len(args) == 0`?
Also, would you like to add an InputError like the one you added for species for `len(args) > 1`?